docs/2_1_common_api.adoc

```diff
@@ -885,6 +940,40 @@ _Example:_
 _A clocked PI controller can be defined as an FMU and the input and output signal of the controller can be defined to be on the same <<inputClock>>._
 _In such a case, the environment deduces the <<clock>> ticks from the components that are connected to the PI controller (e.g. a component connected to the input, or to the output may define the relevant <<clock>>).]_
 
+===== Clock Unions [[clock-relationships-for-communication-point-clocks]]
+
+It is possible to define a dependency from an <<outputClock>> to an <<inputClock>> of the same FMU using the <<triggeredBy>> attribute of <<inputClock>> variables.
+An <<outputClock,`output clock`>> and the associated <<inputClock,`input clocks`>> define a clocks union to indicate that the <<outputClock>> triggers the associated <<inputClock, `input clocks`>> at the same time instant.
+It is only meaningful to combine an <<outputClock,`output clock`>> with an <<periodic,aperiodic>> <<inputClock>>.
```

Comment:
periodic --> aperiodic:
`It is only meaningful to combine an <<outputClock,`output clock`>> with an <<aperiodic,aperiodic>> <<inputClock>>`